carelink.js

```diff
@@ -35,13 +35,25 @@ var Client = exports.Client = function (options) {
   var CARELINKEU_TOKEN_COOKIE = 'auth_tmp_token';
   var CARELINKEU_TOKENEXPIRE_COOKIE = 'c_token_valid_to';
 
+  var CARELINK_ME_URL = 'https://' + carelinkServerAddress + '/patient/users/me';
+  var CARELINK_COUNTRY_SETTINGS_URL = 'https://' + carelinkServerAddress + '/patient/countries/settings?countryCode='+(options.countrycode || DEFAULT_COUNTRYCODE)+'&language='+(options.lang || DEFAULT_LANGCODE);
   var CARELINK_SECURITY_URL = 'https://' + carelinkServerAddress + '/patient/j_security_check';
   var CARELINK_AFTER_LOGIN_URL = 'https://' + carelinkServerAddress + '/patient/main/login.do';
   var CARELINK_JSON_BASE_URL = 'https://' + carelinkServerAddress + '/patient/connect/ConnectViewerServlet?cpSerialNumber=NONE&msgType=last24hours&requestTime=';
   var CARELINK_LOGIN_COOKIE = '_WL_AUTHCOOKIE_JSESSIONID';
   var user_agent_string = [software.name, software.version, software.bugs.url].join(' // ');
 
-  var carelinkJsonUrlNow = function () {
+  var carepartner = false;
```

Comment:
Is this a module global, or captured in a closure or configuration object?  Is it possible to avoid a global to make this re-entrant/safe for multitenant/batch use?

Reply:
@bewest, thanks for your review!
I have refactored a bit in my newest commit so this shouldn't be an issue.